framework/core/locale/core.yml

```diff
@@ -700,13 +704,18 @@ core:
     modal:
       close: Close
 
+    # These translations are used in multi-select components.
+    multi_select:
+      placeholder: Select multiple options
+
     # These translations are used in the navigation header.
     nav:
       drawer_button: Open Navigation Drawer
 
     # These translations are used in forum & admin notices.
     notices:
-      maintenance_mode_2: Down for maintenance. Only administrators may access the forum.
+      maintenance_mode_2: Down for maintenance. Only administrators can access the forum.
```

Comment:
similar here, why not use the human readable names instead of the numbers? Then we'd only have to sync numbers in the backend + frontend enums.